### DISCUSSION TITLE
Using translation keys with double colon does not work

### DISCUSSION THREAD

**Discussion by renepardon** [spatie/laravel-translation-loader#144] [Bugs]
Hi guys,

I import the translations like this:

<img width="635" alt="Bildschirmfoto 2022-02-25 um 17 43 47" src="https://user-images.githubusercontent.com/945323/155753736-b2b1d052-7ec1-4d39-b495-6a1ddab98407.png">

"pages" in this case is a [nwidard/laravel-modules](https://github.com/nWidart/laravel-modules) module. The translations are located within the modules own resource directory. To access such a translation, I would usually use `__('pages::general.labels.pages_admin_languages')` where "pages" is the module and "general" the name of the file.

If I change the group key to "general" only, without the "pages::" prefix, the translation gets loaded from DB when accessing it like this: `__('general.labels.pages_admin_languages')`. But we really need the namespace/prefix. What can we do here?

**Comment by vahidalvandi** [comment]
so i have same issue. i wan to make editabel packages lang published
